packages/@react-native-windows/cli/src/runWindows/utils/vsInstalls.ts

```diff
@@ -43,7 +43,9 @@ function vsWhere(args: string[], verbose?: boolean): any[] {
     );
   }
 
-  const cmdline = `"${vsWherePath}" ${args.join(' ')} -format json -utf8`;
+  const cmdline = `cmd /c chcp 65001>nul && "${vsWherePath}" ${args.join(
```

Comment:
nice :) chcp bring back memories of the 90s and DOS.

Can you specify paths for cmd and chcp? they should be in system32:
```suggestion
  const system32 = `${process.env.SystemRoot}\\System32`;
  const cmdline = `${system32}\\cmd.exe /c ${system32}\\chcp.com 65001>nul && "${vsWherePath}" ${args.join(
```